docs/content/guides/dagster/branch_deployments.mdx

```diff
@@ -178,27 +178,40 @@ Refer to the [Dagster Cloud environment variables documentation](/dagster-cloud/
 
 We’ll first need to define a job that clones our `PRODUCTION` database for each branch deployment. Later, in our GitHub actions workflow, we can trigger this job to run upon each redeploy. Each clone will be named `PRODUCTION_CLONE_<ID>` with `<ID>` representing the pull request ID, ensuring each branch deployment has a unique clone. This job will drop a database clone if it exists and then reclone from production, ensuring each redeployment has a fresh clone of `PRODUCTION`:
 
+<Note>
+  Why use ops and jobs instead of assets? We'll be writing ops to clone the
```

Comment:
```suggestion
  <strong>Why use ops and jobs instead of assets?</strong> We'll be writing ops to clone the
```

I think this explanation is fine. Suggesting we bold the first sentence to make the point of the callout skimmable :)